hack/release.toml

```diff
@@ -29,6 +29,8 @@ xfsprogs: 6.15.0
 systemd-udevd and systemd-boot: 257.7
 lvm2: 2.03.33
 cryptsetup: 2.8.0
+e2fsprogs: 1.47.3
+NVIDIA production drivers: 570.158.01
```

Comment:
unsure if we add it here

Reply:
let's not go too much into details here, NVIDIA is only part of the extensions, and e2fsprogs is a minor thing